E-commerce/index.html

```diff
@@ -11,59 +11,68 @@
 </head>
 
 <body>
-  <!-- Nav Bar -->
-  <div class="topnav">
-    <a href="index.html">Home</a>
-    <a href="products.html">Products</a>
-    <a href="contact.html">Contact</a>
-  </div>
-  <!-- Hero Image -->
-  <div class="hero-image"></div>
-  <!-- Best Selling Section -->
-  <main>
-    <section class="best-selling">
-      <h1>Best Selling</h1>
-      <p>Shop our customers favorite items</p>
-      <div class="grid-container">
-        <div class="flex"><img
-            src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTEnzhBTtcJOAlFHY2WtkpBf7oIdM9MLqaSDg&usqp=CAU"
-            alt="laser engraved box">
-          <h2>Laser Engraved Box</h2>
-          <p>Price: $10</p>
+<!-- Nav Bar -->
+<div class="topnav">
+  <a href="index.html">Home</a>
+  <a href="products.html">Products</a>
+  <a href="contact.html">Contact</a>
+</div>
+<!-- Hero Image -->
+<div class="hero-image"></div>
+<!-- Best Selling Section -->
+<main>
+  <section class="best-selling">
+    <h1>Best Selling</h1>
+    <p>Shop our customers favorite items</p>
+    <div class="grid-container">
+      <div class="flex">
+        <div class="img-container">
+          <img
+                  src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTEnzhBTtcJOAlFHY2WtkpBf7oIdM9MLqaSDg&usqp=CAU"
+                  alt="laser engraved box">
         </div>
```

Comment:
Separating the image and div tags makes these easier to read, and more identifiable to what contains what, which will help when debugging or if someone is new to your code!